lib/playlistGenerator/PlaylistGenerator.js

```diff
@@ -50,8 +50,7 @@ function PlaylistGenerator (entryObject) {
     that.startPromise = null;
     that.persistPromise = qResolved;
     that.playListLimits =  {
-        manifestTimeWindowInMsec: Math.ceil(entryObject.playWindow * 1000 * hlsStoreFilefactor),
-        maxClipCountPerPlaylist: Math.max(10,entryObject.maxChunkCount)
+        manifestTimeWindowInMsec: Math.ceil(entryObject.playWindow * 1000 * hlsStoreFilefactor)
```

Comment:
please run grunt->unit-test->dev
you can modify grunt-config.js: 
unit_tests : 'tests/unit/*' => unit_tests : 'tests/unit/PlaylistGenerator-spec.js' 
to limit unit testing only to playlistGenerator stuff